### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by lazyCode-404**
### Things implemented in this milestone:
- Added styling to API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-# Leaderboard-Project
\ No newline at end of file
+# LeaderBoard
+
```
Comment (tufoinnkuo10) [inline README.md:2]: -[ ] Great job so far. The wireframe looks ok, styling is on point. please kindly add a good description to the readme about the app and how it functions.  It should have instructions about where and how to generate any necessary keys and credentials, how to set, store, and generate them

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @BwalyaNonde 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ board updates automatically after refresh
✅ submit works well
✅ refresh works well

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (lazyCode-404) [reply README.md:2]: Thank you so much for the review, I will implement the changes advised.

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @BwalyaNonde 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**